### ISSUE TITLE
Task: Mack LA Export Configurator Launch Effort

### ISSUE THREAD

**Issue by ZoeBeykirch** [hlxsites/vg-macktrucks-com-co#18]
### Description
Leveraging the work that has been done for the [Mexico market configurator](https://github.com/hlxsites/vg-macktrucks-com/issues/558) and the [spike investigation](https://github.com/hlxsites/vg-macktrucks-com/issues/790), this ticket is to establish the configurator launch for the rest of the LA market sites for Anthem and Granite models. 

### Acceptance Criteria for Front End

- Configure the configurator for market sites, ensuring the language and emission standards are correct for Anthem and Granite:
- [ ] Argentina
- [ ] Bahamas
- [ ] Bolivia
- [ ] Chile
- [ ] Colombia
- [ ] Costa Rica
- [ ] Dominican Republic
- [ ] Ecuador
- [ ] El Salvador 
- [ ] Guatemala
- [ ] Guyana
- [ ] Haiti
- [ ] Honduras
- [ ] Nicaragua
- [ ] Panama
- [ ] Peru
- [ ] Trinidad & Tobago
- [ ] Venezuela

### Reference 
**Markets for configurator:** 
| Country | Language | Locale | Emissions Options |
|----------|----------|----------|----------|
| 1. Argentina | Spanish | es-AR | Euro 3, Euro 5 | 
| 2. Bahamas | English | en-BS | Euro 3, Euro 5 |
| 3. Bolivia | Spanish | es-BO | Euro 3, Euro 5 |
| 4. Chile | Spanish | es-CL | Euro 5, GHG|
| 5. Colombia | Spanish | es-CO | Euro 5, GHG |
| 6. Costa Rica | Spanish | es-CR | Euro 3, Euro 5 |
| 7. Dominican Republic | Spanish | es-DO | Euro 3, Euro 5 |
| 8. Ecuador | Spanish | es-EC |  Euro 3, Euro 5 |
| 9. El Salvador | Spanish | es-SV | Euro 3, Euro 5 |
| 10. Guatemala | Spanish | es-GT | Euro 3, Euro 5 |
| 11. Guyana | English | en-GY | Euro 3, Euro 5 |
| 12. Haiti | English | en-HT | Euro 3, Euro 5|
| 13. Honduras | Spanish | es-HN | Euro 3, Euro 5 |
| 14. Nicaragua | Spanish | es-NI | Euro 3, Euro 5 |
| 15. Panama | Spanish | es-PA |  Euro 3, Euro 5 |
| 16. Peru | Spanish | es-PE | Euro 3, Euro 5 |
| 17. Trinidad & Tobago | English | en-TT |  Euro 3, Euro 5  |
| 18. Venezuela | Spanish | es-VE | Euro 3, Euro 5 |

**Comment by mollyaljenkins** [comment]
On hold until content work is done

**Comment by Kannan-CM** [comment]
Links for testing:

AR:

https://main--vg-macktrucks-com-ar--hlxsites.hlx.page/trucks/anthem/build/#/anthem
https://main--vg-macktrucks-com-ar--hlxsites.hlx.page/trucks/granite-series/build/#/granite

**Comment by Kannan-CM** [comment]
BO:
https://main--vg-macktrucks-com-bo--hlxsites.hlx.page/trucks/anthem/build/#/anthem
https://main--vg-macktrucks-com-bo--hlxsites.hlx.page/trucks/granite-series/build/#/granite

CL:
https://main--vg-macktrucks-cl--hlxsites.hlx.page/trucks/anthem/build/#/anthem
https://main--vg-macktrucks-cl--hlxsites.hlx.page/trucks/granite-series/build/#/granite

CO:
https://main--vg-macktrucks-com-co--hlxsites.hlx.page/trucks/anthem/build/#/anthem
https://main--vg-macktrucks-com-co--hlxsites.hlx.page/trucks/granite-series/build/#/granite

CR:
https://main--vg-macktrucks-cr--hlxsites.hlx.page/trucks/anthem/build/#/anthem
https://main--vg-macktrucks-cr--hlxsites.hlx.page/trucks/granite-series/build/#/granite

DO:
https://main--vg-macktrucks-com-do--hlxsites.hlx.page/trucks/anthem/build/#/anthem
https://main--vg-macktrucks-com-do--hlxsites.hlx.page/trucks/granite-series/build/#/granite

EC:
https://main--vg-macktrucks-com-ec--hlxsites.hlx.page/trucks/anthem/build/#/anthem
https://main--vg-macktrucks-com-ec--hlxsites.hlx.page/trucks/granite-series/build/#/granite

**Comment by Kannan-CM** [comment]
GT:
https://main--vg-macktrucks-gt--hlxsites.hlx.page/trucks/anthem/build/#/anthem
https://main--vg-macktrucks-gt--hlxsites.hlx.page/trucks/granite-series/build/#/granite

HN:
https://main--vg-macktrucks-hn--hlxsites.hlx.page/trucks/anthem/build/#/anthem
https://main--vg-macktrucks-hn--hlxsites.hlx.page/trucks/granite-series/build/#/granite

NI:
https://main--vg-macktrucks-com-ni--hlxsites.hlx.page/trucks/anthem/build/#/anthem
https://main--vg-macktrucks-com-ni--hlxsites.hlx.page/trucks/granite-series/build/#/granite

PA:
https://main--vg-macktrucks-com-pa--hlxsites.hlx.page/trucks/anthem/build/#/anthem
https://main--vg-macktrucks-com-pa--hlxsites.hlx.page/trucks/granite-series/build/#/granite

PE:
https://main--vg-macktrucks-com-pe--hlxsites.hlx.page/trucks/anthem/build/#/anthem
https://main--vg-macktrucks-com-pe--hlxsites.hlx.page/trucks/granite-series/build/#/granite

VE:
https://main--vg-macktrucks-com-ve--hlxsites.hlx.page/trucks/anthem/build/#/anthem
https://main--vg-macktrucks-com-ve--hlxsites.hlx.page/trucks/granite-series/build/#/granite

**Comment by Kannan-CM** [comment]
TT:
https://main--vg-macktrucks-tt--hlxsites.hlx.page/trucks/anthem/build/#/anthem
https://main--vg-macktrucks-tt--hlxsites.hlx.page/trucks/granite-series/build/#/granite

HT:
https://main--vg-macktrucks-ht--hlxsites.hlx.page/trucks/anthem/build/#/anthem
https://main--vg-macktrucks-ht--hlxsites.hlx.page/trucks/granite-series/build/#/granite

GY:
https://main--vg-macktrucks-com-gy--hlxsites.hlx.page/trucks/anthem/build/#/anthem
https://main--vg-macktrucks-com-gy--hlxsites.hlx.page/trucks/granite-series/build/#/granite

**Comment by Kannan-CM** [comment]
Argentina now Shows Euro 3 on Granite.

**Comment by Kannan-CM** [comment]
EL Salvador is also fixed:
https://main--vg-macktrucks-sv--hlxsites.hlx.page/trucks/granite-series/build/#/granite
https://main--vg-macktrucks-sv--hlxsites.hlx.page/trucks/anthem/build/#/anthem

**Comment by Kannan-CM** [comment]
Where can I find "Your unique Mack"?

**Comment by silviulungu** [comment]
<img width="1721" alt="Screenshot 2024-10-24 at 11 29 44" src="https://github.com/user-attachments/assets/552e283b-f127-454c-b485-a057777ca240">

**Comment by Kannan-CM** [comment]
Thanks for the Screenshot @silviulungu ,

That woudl be for Rapid Images to fix.

**Comment by alexiscoelho** [comment]
As stated in:
https://github.com/hlxsites/vg-macktrucks-com-mx/issues/23

> This issue is simply fixed by content.
The truck-configurator url metadata needs to be configured like:
/truck-configurator.json
instead of
https://www.macktrucks.com.ar/truck-configurator.json

**Comment by Kannan-CM** [comment]
HI @silviulungu ,

This is completed

**Comment by silviulungu** [comment]
The issue has been retested and here are the observations:

1. All of the issues from my previous comment are not fixed.

2. "Euro 5" emission is present on Peru market, on Granite. There must be only "Euro 3".
![Image](https://github.com/user-attachments/assets/938884fd-7a45-41b6-a62f-cc49b47746c5)

3. "Euro 5" emission is present on Guatemala market, on Anthem. There must be only "Euro 3".
![Image](https://github.com/user-attachments/assets/87dec466-417f-4863-b51a-3cf6a7602e20)

4. "Euro 5: emission is present on Venezuela market, on Anthem. There must be only "Euro 3".
![Image](https://github.com/user-attachments/assets/57a8e5f7-432f-4864-adc0-266e0a8932c8)

**Comment by Kannan-CM** [comment]
All the above are fixed:

Sometimes it might be privew issue.

in case you see the reload button is highlighted. Could you please click it? So the updated content will appear?
![image](https://github.com/user-attachments/assets/0e8ce76a-7e1d-4fe8-8c8f-c5baa04625c4)

**Comment by Kannan-CM** [comment]
after clicking the reload button:
![image](https://github.com/user-attachments/assets/72d9eb0b-a1d7-4410-aac3-8950b0a3d706)

**Comment by Varunvolvo** [comment]
The Configurator links for Anthem and Granite have been updated for all the Export markets (LA markets).  All are saved in draft and good for preview, which can be published once the testing is done.

**Comment by Kannan-CM** [comment]
And here are the links for testing:

https://main--vg-macktrucks-bs--hlxsites.hlx.page/nav
https://main--vg-macktrucks-com-do--hlxsites.hlx.page/nav
https://main--vg-macktrucks-com-pa--hlxsites.hlx.page/nav
https://main--vg-macktrucks-com-ve--hlxsites.hlx.page/nav
https://main--vg-macktrucks-cr--hlxsites.hlx.page/nav
https://main--vg-macktrucks-gt--hlxsites.hlx.page/nav
https://main--vg-macktrucks-hn--hlxsites.hlx.page/nav
https://main--vg-macktrucks-com-gy--hlxsites.hlx.page/nav
https://main--vg-macktrucks-com-ni--hlxsites.hlx.page/nav
https://main--vg-macktrucks-sv--hlxsites.hlx.page/nav
https://main--vg-macktrucks-tt--hlxsites.hlx.page/nav
https://main--vg-macktrucks-cl--hlxsites.hlx.page/nav
https://main--vg-macktrucks-com-ar--hlxsites.hlx.page/nav
https://main--vg-macktrucks-com-bo--hlxsites.hlx.page/nav
https://main--vg-macktrucks-com-co--hlxsites.hlx.page/nav
https://main--vg-macktrucks-com-ec--hlxsites.hlx.page/nav
https://main--vg-macktrucks-com-pe--hlxsites.hlx.page/nav
https://main--vg-macktrucks-ht--hlxsites.hlx.page/nav

**Comment by silviulungu** [comment]
Observations remaining:

1. Problems receiving the mail after finishing the configuration
2. The link with the truck configuration inside the mail is not working.
3. The QR code link is redirecting the user at the beginning of the configuration instead of showing his personal truck configuration that he just finished.

As i talked with @Kannan-CM , @Varunvolvo will fix the links above in order to move forward with the testing. Thank you!
